secrets/observability-cloudwatchlogs-config.yaml.sample

```diff
@@ -20,3 +20,11 @@ k8s_credentials_secret_name: #fill here
 # Accepted values: ["openshift-logging"]
 # Default value: "openshift-logging"
 k8s_credentials_secret_namespace: #fill here
+
+aws_enterprise_iam_credentials:
```

Comment:
If possible let's call it `enterprise_aws_iam_credentials` or `aws_iam_credentials_enterprise`: iam credentials are an aws concept not an enterprise concept.

Reply:
:+1: